config/main.py

```diff
@@ -86,11 +88,22 @@
 asic_type = None
 
 #
-# Breakout Mode Helper functions
+# Helper functions
 #
 
+# Sort nested dict
+def sort_dict(data):
+    """Sort of data dict"""
```

Comment:
Docstring is not very informative. Please add a bit more detail of how it sorts.